common/lib/transport/connectionmanager.js

```diff
@@ -66,6 +68,8 @@ var ConnectionManager = (function() {
 			params.echo = 'false';
 		if(this.format !== undefined)
 			params.format = this.format;
+		if(this.stream !== undefined)
```

Comment:
I'm not quite clear what you want. It needs to be set as a connect params somewhere. If not in `getConnectParams`, we could do it in `CometTransport#connect` after the `getConnectParams` call [here](https://github.com/ably/ably-js/blob/master/common/lib/transport/comettransport.js#L47) if you prefer.

Or do you mean you want it added directly into the jsonp Request object, ie [here](https://github.com/ably/ably-js/blob/master/browser/lib/transport/jsonptransport.js#L65)? If so, not sure I agree - that will let us remove it from getConnectParams, but it means specifying it in two places (it still needs to be an attribute on the Transport as well as that's what the requestMode looks at), which isn't very DRY. Or am I misunderstanding?